docs/ref/contrib/admin/index.txt

```diff
@@ -2886,6 +2886,23 @@ Templates can override or extend base admin templates as described in
     Raises ``django.contrib.admin.sites.NotRegistered`` if a model isn't
     already registered.
 
+.. method:: AdminSite.get_log_entries(request)
+
+    Returns a queryset with all entries in model
+    :class:`~django.contrib.admin.models.LogEntry`. This method can be used as
+    hook to filter log entries by models registered in the current site.
+    For example::
+
+        def get_log_entries(self, request):
+            from django.contrib.contenttypes.models import ContentType
+
+            log_entries = super().get_log_entries(request)
+            return log_entries.filter(
+                content_type__in=ContentType.objects.get_for_models(
+                    *self._registry.keys()
+                ).values()
```

Comment:
This might be iffy? It shows up how to narrow the data, which is good, but it exposes `_registry` which AFAIK/IIRC is still basically private implementation API?[^1] I could be wrong and just not recalling where in the docs it might be mentioned...

Flagging it even if I'm wrong, just so it's not glossed over.

[^1]: I don't have a better alternative to suggest, though. _Maybe_ something using `get_app_list`...?

Reply:
Good point, yes! 

I think we could likely just drop the example. It's niche enough as it is. Folks who want this will be able to come up with their own ideas.

Reply:
and how about adding a method/property `registered_models` to class `AdminSite`? This could be useful for all kind of other purposes.

Reply:
@jrief — I think that would need to be separately justified. Just expanding the API for a _maybe_ use-case is not wise. Dropping the example should let us resolve this now. If you think **more** API is needed you can follow-up in a separate ticket.

Reply:
@carltongibson example has been removed.

However, I would like to keep the unit test which implements the (now undocumented) hooking purpose.